package.json

```diff
@@ -6,18 +6,19 @@
   "directories": {
     "test": "test"
   },
+  "peerDependencies": {
+    "@theforeman/vendor": ">= 3.3.2"
+  },
   "dependencies": {
-    "@theforeman/vendor": "^1.7.0",
     "react-json-tree": "^0.11.0"
   },
   "devDependencies": {
-    "@theforeman/vendor-dev": "^1.7.0",
-    "babel-eslint": "^8.2.1",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-react": "^6.24.1",
+    "@babel/core": "^7.7.0",
+    "@theforeman/env": "^3.3.2",
+    "@theforeman/builder": "^3.3.2",
```

Comment:
This will still fail to build since builder is 4 in the packaging already. Please update @sharvit